rskj-core/src/main/java/co/rsk/peg/BridgeMethods.java

```diff
@@ -613,6 +613,61 @@
             (BridgeMethodExecutorTyped) Bridge::registerFastBridgeBtcTransaction,
             activations -> activations.isActive(RSKIP176),
             false
+    ),
+    GET_BEST_BLOCK_HEADER(
+            CallTransaction.Function.fromSignature(
+                    "getBestBlockHeader",
+                    new String[0],
+                    new String[]{"bytes"}
+            ),
+            fixedCost(20000L),
+            (BridgeMethodExecutorTyped) Bridge::getBestBlockHeader,
+            activations -> activations.isActive(RSKIP220),
+            false
```

Comment:
```suggestion
            true
```
Same for the other methods, they are `view` type functions

Reply:
Since you are at it, you could also change it for `GET_ACTIVE_FEDERATION_CREATION_BLOCK_HEIGHT` that I just realized should also have the same value

Reply:
OK, I will change it, but why the parameter is called `onlyAllowsLocalCalls`? A view method could be invoked inside a onchain transaction, ie, invoked from other smart contract

Reply:
You are right, my bad. Didn't realize this method should also be called from contracts. It should go back to `false` then. Sorry for the mix-up

Reply:
OK, no problema, reverted